dist/css/style.css

```diff
@@ -0,0 +1,17 @@
+/* Error: Can't find stylesheet to import.
+ *   ,
+ * 3 | @import "/node_modules/bootstrap/scss/bootstrap.scss";
+ *   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ *   '
+ *   src\scss\style.scss 3:9  root stylesheet */
+
+body::before {
+  font-family: "Source Code Pro", "SF Mono", Monaco, Inconsolata, "Fira Mono",
+      "Droid Sans Mono", monospace, monospace;
+  white-space: pre;
+  display: block;
+  padding: 1em;
+  margin-bottom: 1em;
+  border-bottom: 2px solid black;
+  content: "Error: Can't find stylesheet to import.\a   \2577 \a 3 \2502  @import \"/node_modules/bootstrap/scss/bootstrap.scss\";\d\a   \2502          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\a   \2575 \a   src\\scss\\style.scss 3:9  root stylesheet";
```

Comment:
This is a SASS error

Reply:
how can I resolve this error? for me, it is working just fine even with this error

Reply:
can I just delete this file?